packages/2d/src/scenes/Scene2D.ts

```diff
@@ -117,6 +117,22 @@ export class Scene2D extends GeneratorScene<View2D> implements Inspectable {
     }
   }
 
+  public transformMousePosition(
+    x: number,
+    y: number,
+    size?: {width: number; height: number},
+  ): Vector2 | null {
+    if (size !== undefined) {
+      x -= size.width / 2;
+      y -= size.height / 2;
+    }
+
+    x = Math.round(x);
+    y = Math.round(y);
```

Comment:
Rounding should be done on the `ui` side when displaying the values

Reply:
Rounding was moved to the UI side.